ledger/sandbox-on-x/src/test/it/scala/com/daml/ledger/platform/sandbox/auth/GetAdminUserWithGivenUserIdAuthIT.scala

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.platform.sandbox.auth
+
+import com.daml.ledger.api.v1.admin.user_management_service._
+import io.grpc.{Status, StatusRuntimeException}
+
+import java.util.UUID
+import scala.concurrent.Future
+
+class GetAdminUserWithGivenUserIdAuthIT extends AdminServiceCallAuthTests {
```

Comment:
Is that a misnomer? I don't think we are "getting an admin user" in this test

Reply:
I removed this class